### PR TITLE
SO portal and search box fix

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/views/SO-portal.vue [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    mapbox.addMapboxScript();
+  },
+  mounted() {
+    setTimeout(() => {
+      this.mapShow = true;
+    }, 1000);
+  },
+  methods: {
+    async kyc(data) {
+      console.log("this is me kyc");
```
Comment (arrogant-ant) [inline src/views/SO-portal.vue]: please remove

File: src/components/organisms/o-so-details.vue [hunk 2]
```diff
@@ -0,0 +1,435 @@
+<template>
+  <div class="o_so_details">
```
Comment (arrogant-ant) [inline src/components/organisms/o-so-details.vue]: preferably break this organism.

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

**Comment by arrogant-ant** [comment]
some general feedback:
1. Please use explanatory branch name.
2. Please use one PR for one and only one usecase/ feature/ bug.

File: src/components/molecules/m-contact-details.vue [hunk 3]
```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    <label class="is-size-5 is-size-6-mobile"
+      >Contact Details <span style="color: red">*</span> :</label
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/molecules/m-contact-details.vue:4]: done

File: src/components/molecules/m-contact-details.vue [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+export default {
+  components: { atomInput },
+  name: "m-contact-details",
+  props: {
+    fullName: String,
+    email: String,
+    mno: Number,
+  },
+  data() {
+    return {
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/molecules/m-contact-details.vue:47]: done

File: src/components/molecules/m-search-box.vue [hunk 5]
```diff
@@ -78,7 +77,7 @@ export default {
       animation: "tada",
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/molecules/m-search-box.vue:77]: magnifying glass animation

File: src/components/organisms/o-so-details.vue [hunk 6]
```diff
@@ -0,0 +1,435 @@
...
+    <div class="ps_center">
+      <atom-b-title class="is-size-3 is-size-5-mobile" :text="title" />
+    </div>
+    <br />
+    <form v-on:submit.prevent="onSubmit">
+      <!-- contact details -->
+      <label class="is-size-5 is-size-6-mobile"
+        >Contact Details <span style="color: red">*</span> :</label
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/o-so-details.vue]: done

File: src/components/organisms/o-so-details.vue [hunk 7]
```diff
@@ -0,0 +1,435 @@
...
+          <atom-input
+            v-model="userForm.mno"
+            class="input"
+            :placeholder="contact.mno"
+            :required="required"
+            :types="contact.mobileType"
+          />
+        </div>
+      </div>
+      <!-- <m-contact-details
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/o-so-details.vue]: SO page is in testing phase ..will test in production then after successfull testing will format it .

File: src/components/organisms/o-so-details.vue [hunk 8]
```diff
@@ -0,0 +1,435 @@
...
+        <div class="column">
+          <atom-input
+            v-model="userForm.city"
+            class="input"
+            :placeholder="address.City"
+            :required="required"
+          />
+        </div>
+        <div class="column">
+          <!-- <atom-input
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/o-so-details.vue]: same as above.

File: src/components/organisms/o-so-details.vue [hunk 9]
```diff
@@ -0,0 +1,435 @@
...
+  },
+  methods: {
+    async register() {
+      const res = await fetch("https://maya.parkspot.in/auth/register", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          UserName: "dummy_test3",
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/o-so-details.vue]: this is just for testing right now.

File: src/components/organisms/o-so-details.vue [hunk 10]
```diff
@@ -0,0 +1,435 @@
...
+        ownershipFile: this.ownershipFile,
+        ownerhipPicked: this.ownershipPicked,
+        checkedAmenities: this.checkedAmenities,
+        nofspotDefault: this.nofspotDefault,
+        durationDefault: this.durationDefault,
+        rent: this.rent,
+        terms: this.terms,
+      };
+      console.log("this is the all the data");
+      console.log(aggrData);
```
[the remark this answers is not in the document]
Comment (alias01) [reply src/components/organisms/o-so-details.vue]: ok

File: src/views/SO-portal.vue (hunk 1 above)
Comment (alias01) [reply src/views/SO-portal.vue]: will do after successful testing.

File: src/components/organisms/o-so-details.vue (hunk 2 above)
Comment (alias01) [reply src/components/organisms/o-so-details.vue]: same as above

**Pull request merged**